charts/rancher-eks-operator/1.0.101/templates/NOTES.txt

```diff
@@ -0,0 +1,4 @@
+You have deployed the Rancher EKS operator
+Version: {{ .Chart.AppVersion }}
+Description: This operator provisions EKS clusters
+from EKSClusterConfig CRs.
```

Comment:
Perhaps there's a rule I don't know; is this line wrapping intentional?

Reply:
Probably to be compatible with default printer column widths for commands such as `helm search repo`: https://helm.sh/docs/helm/helm_search_repo/#options